ansible/README.md

```diff
@@ -14,6 +14,8 @@ sudo pip install ansible==2.0.2.0
 
 #### Vagrant users
 Nothing to be done, Ansible is already installed during vagrant provisioning.
+You can skip setup and prereq steps as those have been done by vagrant for you.  
+You may jump directly to [Deploying Using CouchDB](#deploying-using-couchdb)
 
 #### Mac users
 It is assumed that a VM has been provisioned using **docker-machine**.
```

Comment:
Please let's document how an user can setup his machine with `whisk` docker-machine and any other requirements. 

It doesn't look correct that we tell people about using docker-machine on mac when we don't have a good [tools/mac/README](https://github.com/openwhisk/openwhisk/blob/master/tools/macos/README.md) explaining what  the user can do setup his environment like installing docker 1.9.1 and not the latest and other things.

For example:

# Install Docker/Java/Scala

1. brew cask install virtualbox
2. brew install docker
3. brew install docker-machine
4. brew install ant
5. brew install ant-contrib
6. brew cask install java
7. brew install scala
8. brew install gradle
9. sudo pip install jsonschema

# Create and Configure Docker Machine

**Create machine:**

  - docker-machine create -d virtualbox --virtualbox-memory 4096 --virtualbox-boot2docker-url=https://github.com/boot2docker/boot2docker/releases/download/v1.9.1/boot2docker.iso whisk
  - docker-machine ssh whisk

**And then inside the whisk machine, do:**

  - echo DOCKER_TLS=no |sudo tee -a /var/lib/boot2docker/profile > /dev/null
  - echo DOCKER_HOST=\'-H tcp://0.0.0.0:2376\' |sudo tee -a /var/lib/boot2docker/profile > /dev/null
  - echo sudo iptables -t nat -A PREROUTING -p tcp --dport 4243 -j REDIRECT --to-port 2376 |sudo tee -a /var/lib/boot2docker/profile > /dev/null
  - echo /sbin/syslogd |sudo tee /var/lib/boot2docker/bootsync.sh > /dev/null
  - sudo chmod +x /var/lib/boot2docker/bootsync.sh

**Log out of the whisk machine ssh:**

  - eval $(docker-machine env whisk)

**Restart the docker machine:**

  - docker-machine restart whisk

# Getting Information about the Docker Image

Get the machine IP and PORT number:

  - docker-machine url whisk

# Set Docker Machine IP and PORT information

**Update the ~/.bash_profile with the following line**

  - export DOCKER_HOST="tcp://192.168.99.100:2376”

**Source the bash_profile**

  - source ~/.bash_profile